docs/mercure.md

```diff
@@ -8,5 +8,6 @@ No JS library or SDK required!
 ![Mercure](mercure-hub.png)
 
 To enable the Mercure hub, update the `Caddyfile` as described [on Mercure's site](https://mercure.rocks/docs/hub/config).
+The URL to send data is `http://php/.well-known/mercure`, the public URL for clients is at the path `/.well-known/mercure`.
```

Comment:
The `php` domain is only for Symfony Docker and the like. I'm not sure if it's a good idea to use it without more explanations in FrankenPHP's docs. WDYT?

Reply:
oh, i see. kinda forgot the scope of this repo. would something like this be better:

```suggestion
The path to send data as well as the path of the public URL for clients is `/.well-known/mercure`.  When running FrankenPHP inside Docker, the full send URL would look like `http://php//.well-known/mercure` (with `php` being the name of the container running FrankenPHP).
```

Reply:
is sending available from anywhere and only secured by the JWT or also IP restricted to only be possible from within the Docker container?

Reply:
By default, it's only by JWT, but Caddy supports adding all kinds of restrictions, including by IP: https://caddyserver.com/docs/caddyfile/matchers